src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let clone = { ...state };
+  const result = [];
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(clone, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key of action.keysToRemove) {
+          delete clone[key];
+        }
+        break;
+
+      case 'clear':
+        clone = {};
+        break;
+
+      default:
+        break;
```

Comment:
![image](https://user-images.githubusercontent.com/59209337/211486935-324618ce-00c2-43af-b6c3-bfda156f013e.png)

Reply:
Hi, I'm not quite sure what I'm supposed to fix

Reply:
default case should handling error, and you just make it empty
use `throw new Error('your error text')`